app/src/main/java/com/example/parkingsystemkotlin/database/ParkingSpaceReservationDB.kt

```diff
@@ -26,4 +26,12 @@ object ParkingSpaceReservationDB {
             }
         }
     }
+
+    fun getAllReservations(): MutableList<Reservation> {
```

Comment:
do you need this list to be mutable?

Reply:
I changed it to return a List instead of mutableList, thanks!